api.md

```diff
@@ -363,42 +364,25 @@ Etag: "e0b2816b418b3f266309d94426ac7668ab3c1fa87798785bf82f1085cc2f6d9c"
 }
 ```
 
-<a name=tailnet-acl-post></a>
```

Comment:
Ah so I think you want to keep this section around - It looks like there's a `[POST tailnet ACL]` up above that is meant to link to this portion.

Reply:
The new `tailnet-acl-preview-post` section looks good, I think we just want to make sure we keep the `tailnet-acl-post` one around too.